docs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tiger-docs",
```

Comment:
this is probably wrong, these are not tiger-docs, also check description, repository.url and homepage fields

Reply:
You are right, I'll fix this. Thank you for catching this! :)